webview/player/player.js

```diff
@@ -32,6 +33,15 @@ function togglePlayPause(toggle = true) {
   }
 }
 
+function setVideoProgress(value) {
+  if (video.paused || video.ended) {
+    video.play();
+    video.currentTime = value * video.duration;
+  } else {
+    video.currentTime = value * video.duration;
+  }
```

Comment:
`video.currentTime = value * video.duration;` is the same in if and else branch and should be moved out to prevent code duplication.

```suggestion
  video.currentTime = value * video.duration;
  if (video.paused || video.ended) {
    video.play();
  }
```